frontend/packages/console-shared/src/components/lists/DetailPropertyList.scss

```diff
@@ -0,0 +1,12 @@
+ul.pf-c-list:not(.pf-m-inline).co-detail-property-list {
+  list-style-type: none;
+  list-style: none;
```

Comment:
don't these two result in the similar thing? Maybe we can use just `list-style`

Reply:
Right, removing.